google-cloud-spanner/src/test/java/com/google/cloud/spanner/OptionsTest.java

```diff
@@ -200,10 +204,13 @@ public void readEquality() {
   @Test
   public void queryOptTest() {
```

Comment:
nit: please don't abbreviate `Options` to `opts`. (I know this was already like this, but if you are to make any other changes, could you please fix this as well).

Reply:
Done.